.vscode/settings.json

```diff
@@ -0,0 +1,5 @@
+{
+   "files.associations": {
+      "*.hip": "cpp"
+   }
+}
```

Comment:
Just a comment: I'm always on the fence about committing these.

Reply:
I actually created this file by hand, it was **not** auto-created.

This is needed to have VSCode recognise the .hip extension as a C++ file. I think anybody working on this project with VSCode will want to have this set...

Reply:
That's correct. But that makes it much more difficult to have differing settings from you. VSCode puts all kinds of stuff in there, and most of them you probably don't want to commit.

Reply:
My experience is that if the file exists already, then VSCode is much more measured in this. It does try to create other files in `.vscode` automatically. But I don't think it would ever modify this file itself without the user asking for it.

I see your point with this, but until the project gets more complicated, when the users may indeed want to have different settings in their own clones, I thought it would be safe enough to add this simple setting to the repository.

Reply:
So, how strongly do you feel about this? As from my side I'm ready with this PR at this point.

Reply:
Hadn't seen this, not strongly anyway.